### DISCUSSION TITLE
REST API modules query parameter module_bay

### DISCUSSION THREAD

**Discussion by mdoerr2s** [netbox-community/netbox#9393] [Q&A] [answered]
I was trying to query the api for a module in a specific module_bay, but noticed there is no parameter in the REST API (dcim/modules/) for this. 

Since i checked the data model in the nbshell and the model for module provides this information, is there a specific reason, why it is not included in the API?

Since also the module_bay can not be queried for installed devices, is there another way to query module for specific module_bays? Am I missing something?

**Comment by jeremystretch** [comment, marked as answer]
> is there a specific reason, why it is not included in the API?

No; I'd be happy to add it if you'd like to put in a feature request.

**Reply by mdoerr2s** [reply]
Thanks, for the quick reply
